portal/src/graphql/adminapi/UserDetailsAdminActions.tsx

```diff
@@ -0,0 +1,205 @@
+import { Label, Text } from "@fluentui/react";
+import { FormattedMessage } from "@oursky/react-messageformat";
+import React from "react";
+import cn from "classnames";
+import styles from "./UserDetailsAdminActions.module.css";
+import ListCellLayout from "../../ListCellLayout";
+import DefaultButton from "../../DefaultButton";
+import { useSystemConfig } from "../../context/SystemConfigContext";
+import { UserQueryNodeFragment } from "./query/userQuery.generated";
+
+interface DisableUserCellProps {
+  isDisabled: boolean;
+  onDisableData: () => void;
+}
+
+interface AnonymizeUserCellProps {
+  isScheduledAnonymization: boolean;
+  onAnonymizeData: () => void;
+  onCancelAnonymizeData: () => void;
+}
+
+interface RemoveUserCellProps {
+  isScheduledRemoval: boolean;
+  onRemoveData: () => void;
+  onCancelRemoveData: () => void;
+}
+
+const DisableUserCell: React.VFC<DisableUserCellProps> =
+  function DisableUserCell(props) {
+    const { themes } = useSystemConfig();
+    const { isDisabled, onDisableData } = props;
+    return (
+      <ListCellLayout
+        className={cn(
+          styles.cell,
+          styles.actionCell,
+          styles["cell--not-first"]
+        )}
+      >
+        <Text className={cn(styles.cellLabel, styles.actionCellLabel)}>
+          <FormattedMessage id="UserDetailsAdminActions.disable-user.title" />
+        </Text>
+        <Text className={cn(styles.actionCellBody, styles.actionCellLast)}>
+          <FormattedMessage id="UserDetailsAdminActions.disable-user.body" />
+        </Text>
+        {isDisabled ? (
+          <DefaultButton
+            theme={themes.actionButton}
+            className={cn(styles.actionCellActionButton, styles.actionBorder)}
+            iconProps={{ iconName: "Play" }}
+            text={
+              <FormattedMessage id="UserDetailsAdminActions.disable-user.action.enable" />
+            }
+            onClick={onDisableData}
+          />
+        ) : (
+          <DefaultButton
```

Comment:
Introduce a new component AdminActionButton. Handle merging the styles from prop, and the inherit styles of the design of this component. useMergedStylesPlain